knife-tidy override runlist warning (#116)
chef/knife-tidy: README.md

# Summary

> WARNING: If you use `chef-client -o RUNLIST` a lot, this tool is not compatible with that usage. The main way that knife-tidy decides whether it is safe to delete cookbooks is by reading the runlists of all nodes in the chef server. If knife-tidy does not find a certain cookbook or cookbook version in any runlist, it considers that object a deletion candidate. When you regularly use override runlists, those cookbooks are never stored in a node runlist on the chef server. Knife-tidy will never find them and exclude them from deletion.

Reviewer: ```suggestion
WARNING: `knife-tidy` is incompatible with `chef-client -o RUNLIST`.  `knife-tidy` deletes all cookbooks and cookbook versions that are not referenced by node runlists on the Chef Infra Server, even if they are needed by an override runlist.
```
